src/components/AddItem.js

```diff
@@ -1,69 +1,82 @@
 import React, { useState } from 'react';
 import { db } from '../lib/firebase';
-import { useCollection } from 'react-firebase-hooks/firestore';
+import {
+  getToken,
+  words,
+  calculateEstimate,
+} from '@the-collab-lab/shopping-list-utils';
 
 export default function AddItem() {
   const [item, setItem] = useState('');
-  const [price, setPrice] = useState('');
-
-  // Read 1 document
-  // let docRef = db.collection('lists').doc('5');
-  // docRef.get().then((doc) => console.log(doc.data()));
-
-  const [value, loading, error] = useCollection(db.collection('lists'));
-
-  if (!loading) {
-    // console.log(value);
-    console.log(error);
-  }
+  const [urgency, setUrgency] = useState(7);
+  const [userToken, setToken] = useState(getToken()); // sets a new token when component mounts
 
   const handleSubmit = (e) => {
     e.preventDefault();
+    console.log(item, urgency);
     e.target.reset();
     db.collection('lists').add({
```

Comment:
Instead of adding to a collection called `lists`, add it to the collection that uses the users `token`. You'll receive that as a prop.